Makefile

```diff
@@ -83,8 +83,10 @@ endif
 update-1.0-charts: update-remote-maistra-1.0
 	git checkout ${GIT_UPSTREAM_REMOTE}/maistra-1.0 -- ${SOURCE_DIR}/resources/helm/v1.0
 
+.PHONY: update-1.0-templates
+update-1.0-templates:
+	curl -L https://github.com/Maistra/istio-operator/archive/maistra-1.0.tar.gz | tar -xzvC ${SOURCE_DIR}/resources/smcp-templates/v1.0 --strip-components 3 */deploy/smcp-templates
 # XXX: for now, the templates for maistra-1.0 are stored in ./deploy/smcp-templates, so the following won't work
-#.PHONY: update-1.0-templates
 #update-1.0-templates: update-remote-maistra-1.0
 #	git checkout ${GIT_UPSTREAM_REMOTE}/maistra-1.0 -- ${SOURCE_DIR}/resources/smcp-templates/v1.0
 
```

Comment:
Do we still need those comments or they can be removed?

Reply:
I left them there so I can apply them to update-1.1-templates when we get started on 1.2.